### DISCUSSION TITLE
About the types of charts that can be displayed in Dify

### DISCUSSION THREAD

**Discussion by ymiyata910** [langgenius/dify#7131] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

I know that ChartGenerator exists as a tool and that it can display Mermaid diagrams.

Are there any other flow or chart diagrams that can be displayed in Dify?

Thank you.

**Comment by zhujinle** [comment]
In Markdown, use `Mermaid` or `ECharts`
Mermaid can draw flowcharts
ECharts is a feature added in recent versions, which can draw a variety of charts
You only need to output in markdown format, and the front-end components will be automatically rendered without the need for tools.
